### DISCUSSION TITLE
Installation Open RV on Linux Mint

### DISCUSSION THREAD

**Discussion by archct** [AcademySoftwareFoundation/OpenRV#543] [Q&A] [answered]
I'm trying to compile files on linux mint version kernel 5.4.0, but it doesn't work. 
I would like to know if it is possible to install Open RV on my system and if it is possible, I would like to find instructions on how to do it.

**Comment by bernie-laberge** [comment, marked as answer]
Hello @archct,
Unfortunately, Open RV is not currently available in installable binary form. You need to build it.
I have created the following issue so that Linux Mint support might get added by the community:
https://github.com/AcademySoftwareFoundation/OpenRV/issues/580
